package.json

```diff
@@ -19,13 +19,13 @@
     "**/*": "prettier --write --ignore-unknown"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.818fb3956",
-    "@docusaurus/preset-classic": "2.0.0-beta.818fb3956",
+    "@docusaurus/core": "^2.0.0-beta.4",
+    "@docusaurus/preset-classic": "^2.0.0-beta.4",
     "@mdx-js/react": "^1.5.8",
     "clsx": "^1.1.1",
-    "mermaid": "^8.10.1",
```

Comment:
We can get rid of mermaid if you want.
We will be using svgs for diagrams exported from Lucidchart.